README.md

```diff
@@ -45,7 +44,7 @@ def count(items: List[str], item: str) -> int:
     return items.count(item)
 
 # test case
-@pytest.mark.parametrize('case', deal.cases(count))
+@deal.cases(count)
 def test_count(case: deal.TestCase):
     case()
```

Comment:
I have mixed feelings about the short form. Since the function is created inside the deal, the assignment line `test_div = deal.case(...)` is not included into the traceback. I'm considering to patch the traceback frames but it's not so trivial in later Python versions. So, the short form is good only if you have only one test per file.

Reply:
Ah, I see.  That *is* a good reason to use the longer form, though Pytest should print the name you assigned it to as the testid.